pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/RealtimeTableDataManager.java

```diff
@@ -115,6 +115,8 @@ public class RealtimeTableDataManager extends BaseTableDataManager {
 
   private TableDedupMetadataManager _tableDedupMetadataManager;
   private TableUpsertMetadataManager _tableUpsertMetadataManager;
+  // Object to track ingestion delay for all partitions
+  private MaximumPinotConsumptionDelayTracker _maximumPinotConsumptionDelayTracker;
```

Comment:
suggest rename pinotConsumptionDelayTracker (we can omit the maximum)

Reply:
used consumptionDelayTracker as suggested above.